youtube_dl/extractor/parler.py

```diff
@@ -0,0 +1,73 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from urllib import parse
+
+from .common import InfoExtractor
+
+from ..utils import clean_html, unified_timestamp
```

Comment:
Please format like this for mantainability; also, add routines per other suggestions:
```suggestion
from ..utils import (
    clean_html,
    strip_or_none,
    unified_timestamp,
    urlencode_postdata,
)
```